src/modules/search/hooks/useSearch.js

```diff
@@ -34,15 +34,11 @@ export const useSearch = (search = '') => {
   const isLoading =
     delegates.isLoading || transactions.isLoading || addresses.isLoading || blocks.isLoading;
 
-  const isFetched =
-    delegates.isFetched && transactions.isFetched && addresses.isFetched && blocks.isFetched;
-
```

Comment:
So `isFetched` is not needed?
How about if we change it to
```
const isFetched =
     (isDelegate  && delegates.isFetched) || (isTxId  && transactions.isFetched) || (isAddress  && addresses.isFetched) || (isBlockHeight  && blocks.isFetched);
```

Reply:
@reyraa It might be needed but this is another way to handle it